taxcalc/params.json

```diff
@@ -1010,10 +1010,10 @@
         "value":     [0]        
     },
     "_ALD_SelfEmploymentTax_HC":{
-        "long_name": "Deduction for self-employment tax haircut",
-	    "description": "This percentage reduces the employer equivalent portion of self-employment deduction before AGI. ",
-        "irs_ref": "Form , line , ",
-        "notes": "The fil deduction amount would be (1 - Hair Cut) * Self Employment Tax Deduction",
+        "long_name": "Adjustment for self-employment tax haircut",
+	"description": "This percentage would limit the employer equivalent portion of self-employment adjustment if greater than zero ",
```

Comment:
maybe -> "This percentage, if greater than zero, reduces the employer equivalent portion of self-employment adjustment"